apps/zotonic_mod_base/priv/lib/js/modules/z.dialog.js

```diff
@@ -166,8 +166,19 @@
             if (typeof($.widgetManager) != 'undefined') {
                 $dialog.widgetManager();
             }
+
             z_editor_add($dialog);
 
+            setTimeout(function() {
+                $('a.close').focus();
```

Comment:
Should we check if there is an input element with focus?

Reply:
Good one! Yes

Reply:
Fixed!